docs/api.md

```diff
@@ -347,6 +347,10 @@ those common validations are defined here.
 
 * `schema`: {
     * `id: isA.string.length(32).regex(HEX_STRING)
+    * `location`: isA.object({ city: isA.string.optional.allow(null)
```

Comment:
Does someone have the pre-commit hook disabled that is supposed to update these docs?

Reply:
It's not deliberate:

https://github.com/mozilla/fxa-auth-server/issues/2128

Haven't had time to properly dig into it yet, sorry.